dev/test/specs/starter-test.js

```diff
@@ -187,7 +183,7 @@ function openSuggestionDrawer() {
 
     if (!_isSuggestionsDrawerOpen()) {
       console.log('suggestions drawer not opening?');
-      browser.debug();
+      // browser.saveScreenshot(`test/errorShots/suggestions-drawer-closed-${Date.now()}.png`);
```

Comment:
Uncomment? Also did you intent to remove debug above?

Reply:
Hmm, really neither of these lines should be in the codebase. Let me remove the screenshot line too.